src/js/jsx/sections/libraries/LibraryList.jsx

```diff
@@ -96,24 +160,203 @@ define(function (require, exports, module) {
 
-            return (
+            var sharedLibrary = false,
+                libraryLink,
+                shareLink,
+                collaborateLink;
+
+            if (selectedLibrary) {
+                sharedLibrary = selectedLibrary.collaboration !== _REGULAR_LIBRARY;
+                libraryLink = "https://assets.adobe.com/assets/libraries/" + selectedLibrary.id;
+                shareLink = libraryLink + "?dialog=share";
+                collaborateLink = libraryLink + "?dialog=collaborate";
+            }
+
+            return (<div className="libraries__bar__top__content">
                 <Datalist
                     list={listID}
                     className="dialog-libraries"
                     options={listOptions}
                     value={selectedLibraryName}
                     live={false}
-                    onChange={this._handleChange}
+                    autoSelect={false}
+                    onChange={this._handleChangeLibrary}
                     defaultSelected={selectedLibraryID}
                     disabled={this.props.disabled} />
+                <SplitButtonList className="libraries__split-button-list">
+                    <SplitButtonItem
+                        title={strings.TOOLTIPS.LIBRARY_SHARE}
+                        iconId="libraries-collaborate"
+                        className={sharedLibrary && "libraries__split-button-collaborate"}
+                        disabled={!selectedLibrary}
+                        onClick={ui.openURL.bind(null, collaborateLink)} />
+                    <SplitButtonItem
+                        title={strings.TOOLTIPS.LIBRARY_SEND_LINK}
+                        iconId="libraries-share"
+                        disabled={!selectedLibrary}
+                        onClick={ui.openURL.bind(null, shareLink)} />
+                    <SplitButtonItem
+                        title={strings.TOOLTIPS.LIBRARY_VIEW_ON_WEBSITE}
+                        iconId="libraries-viewonsite"
+                        disabled={!selectedLibrary}
+                        onClick={ui.openURL.bind(null, libraryLink)} />
+                </SplitButtonList>
+            </div>);
+        },
+
+        /**
+         * Render text input and buttons for creating a library or renaming the current library.
+         *
+         * @private
+         * @return {?ReactComponent}
+         */
+        _renderLibraryNameInput: function () {
+            var command = this.state.command;
+
+            if (!command || command === _DELETE_LIBRARY) {
+                return null;
+            }
+
+            var rename = command === _RENAME_LIBRARY;
+
+            return (<div className="libraries__bar__top__content libraries__bar__top__content-input">
+                <TextInput
+                    type="text"
+                    live={true}
+                    continuous={true}
+                    className="libraires__bar__input"
```

Comment:
Also, as far as these CSS classes are concerned, `__` should be only used to separate the "states" of a class from each other. So most of these should be `libraries-bar_input`, `libraries-bar_top-content` etc.